v2/elasticsearch-common/pom.xml

```diff
@@ -27,6 +27,7 @@
     </parent>
 
     <artifactId>elasticsearch-common</artifactId>
+    <version>1.0-SNAPSHOT</version>
```

Comment:
Is this file needs to be included as current version in master branch is already 1.0-SNAPSHOT